pkg/services/ngalert/notifier/compat.go

```diff
@@ -0,0 +1,42 @@
+package notifier
+
+import (
+	"encoding/json"
+
+	alertingNotify "github.com/grafana/alerting/notify"
+
+	apimodels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+)
+
+func PostableGrafanaReceiverToGrafanaReceiver(p *apimodels.PostableGrafanaReceiver) *alertingNotify.GrafanaReceiver {
```

Comment:
It takes pointer because we use pointers in our API models. It returns pointers because of the same reason - we use pointers downstream. I think we can change that but maybe in the follow-up PRs.